inst/doc/ValidateModel.Rmd

```diff
@@ -95,8 +120,8 @@ colnames(failures) <- c("Industry", "")
 reportValidationResult(failures)
 ```
 
-#### 4. Sum of each commodity's output across all states must almost equal(<= 0.001) commodity output in US Make Table.
-```{r check4}
+#### 5. Sum of each commodity's output across all states must almost equal(<= 0.001) commodity output in US Make Table.
+```{r check5}
```

Comment:
Rethinking about this validation check (and the next one). Because in the Use table, state imports (F050) is not allocated from US imports, `sum(state_F050)!=US_F050`. As a result, state commodity (summed row-wise from state Use) can't sum up to US commodity output. We need to think of another way to validate state commodity output. @WesIngwersen

Reply:
There is a deeper reason for failures in this check: at state level, `q_make ≠ q_use`, which also causes failures at [normalizing](https://github.com/USEPA/stateior/blob/dev/inst/doc/ValidateModel.Rmd#L429-L435) state Make by CommOutput.
This `q_make ≠ q_use` can be attributed to my explanation above about `F050`, but I'm skeptical about allocating `US_F050` to states would fix it - it may be a similar issue to the state Value Added issue I recently solved and require RAS balancing as well.
Your thoughts? @WesIngwersen

Reply:
I tried to apply RAS to balance state `F050` targeting at `q_make == q_use` and `sum(state_F050) == US_F050`, but cannot achieve the desired results. See detailed explanations in issue #12.